jep/1/README.adoc

```diff
@@ -0,0 +1,939 @@
+of these things during any part of the life of a JEP.
+
+Because the JEPs are maintained as text files in a versioned
+repository, their revision history is the historical record of the
+feature proposal footnoteref:[repo, https://github.com/jenkinsci/jep].
+
+[NOTE]
+====
+For non-text files added to the repository, such as images, the files must be
+readable with free and open source tools, and should ideally be editable by
+other contributors. For example, use SVG rather than PSD for vector graphics.
+====
+
+
+=== JEP Types
+
+There are three kinds of JEP:
+
+. A **Standards Track** JEP describes a new feature or implementation
+ for Jenkins. It may also describe an interoperability or
+ backwards-compatibility standard which will be supported for the feature in
+ current versions of Jenkins, moving forward.
+. An **Informational** JEP describes a Jenkins design issue, or
+ provides general guidelines or information to the Jenkins community,
+ but does not propose a new feature. Informational JEPs do not
+ necessarily represent a Jenkins community consensus or
+ recommendation, so users and implementers are free to ignore
+ Informational JEPs or follow their advice.
+. A **Process** JEP describes a process surrounding Jenkins, or
+ proposes a change to (or an event in) a process. Process JEPs are like
+ Standards Track JEPs but apply to areas other than the Jenkins codebase
+ itself. They may propose an implementation, but not for what would be
+ generally considered the Jenkins codebase; they often require community
+ consensus; unlike Informational JEPs, they are more than recommendations, and
+ users are typically not free to ignore them. Examples include procedures,
+ guidelines, changes to the decision-making process, and changes to the tools
+ or environment used in Jenkins development. Any meta-JEP (JEP focusing on the
+ improvement of the Jenkins Enhancement Proposal process) is also considered a
+ Process JEP.
+
+=== JEP Terminology
+
+==== BDFL
+
+There are several references in this JEP to the "BDFL". This acronym stands for
+"Benevolent Dictator for Life", and refers to
+link:https://github.com/kohsuke[Kohsuke Kawaguchi],
+original creator of Jenkins.
+
+==== Editors
+
+The JEP editors are individuals responsible for managing the administrative
+and editorial aspects of the JEP workflow (e.g. assigning JEP numbers and
+changing their status). See <<editor-responsibilities, JEP Editor Responsibilities & Workflow>> for
+details. The current editors are:
+
+* link:https://github.com/rtyler[R. Tyler Croy]
+* link:https://github.com/oleg-nenashev[Oleg Nenashev]
+* link:https://github.com/bitwiseman[Liam Newman]
+* link:https://github.com/omehegan[Owen Mehegan]
+* link:https://github.com/orrc[Christopher Orr]
+
+JEP editorship is by invitation of the current editors. All of the JEP workflow
+can be conducted via the GitHub JEP Repository footnoteref:[repo] and pull
+requests.
+
+==== Conributor
```

Comment:
Typo, also should be plural for consistency with 'Editors' above?

Reply:
Switching them all to singular. 

RESOLVED.